README.md

```diff
@@ -287,7 +287,10 @@ yotiClient.getActivityDetails(token).then((activityDetails) => {
     * [X] Address `postalAddress`
     * [X] Gender `gender`
     * [X] Nationality `nationality`
-  * [X] Base64 Selfie URI `.getBase64SelfieUri()`
+  * [X] UserProfile `.getProfile()`
+    * [X] Full Name `.getFullName().getValue()`
+    * [X] Full Name - Source `.getFullName().getSources()[0].getValue()`
```

Comment:
Since there can be multiple sources, do you think we should either mention that, or show a way of retrieving all of the sources here? (obviously same below with verifiers)

Reply:
I think we should remove this list altogether and have it properly displayed in the documentation.